src/Controller/UsersController.php

```diff
@@ -499,6 +499,7 @@ public function edit($id = null)
 
             if ($this->request->is(['patch', 'post', 'put'])) {
                 $user = $this->Users->patchEntity($user, $this->request->data);
+                $user->set('gender', 2);
 
```

Comment:
Oui tu as raison :p
Le 11 mai 2016 08:03, "Félix Leblanc" <notifications@github.com> a écrit :

> In src/Controller/UsersController.php
> <https://github.com/MaisonLogicielLibre/Website/pull/1131#discussion_r62835336>
> :
>
> > @@ -499,6 +499,7 @@ public function edit($id = null)
> >
> >              if ($this->request->is(['patch', 'post', 'put'])) {
> >                  $user = $this->Users->patchEntity($user, $this->request->data);
> > +                $user->set('gender', 2);
> >
>
> Tu ne devrais pas plutôt utiliser la variable du post au lieu de hardcoder
> 2?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/MaisonLogicielLibre/Website/pull/1131/files/428a2f07c893be498d09617fea95fb5561e4eec6#r62835336>
>